homeassistant/components/anova_cooker/config_flow.py

```diff
@@ -0,0 +1,49 @@
+"""Config flow for Anova Sous Vide Cooker integration."""
+import logging
+
+import voluptuous as vol
+
+from homeassistant import config_entries, core
+
+from .const import DOMAIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema({"device_id": str})
+
+
+async def validate_input(hass: core.HomeAssistant, data):
+    """Validate the user input allows us to connect.
+
+    Data has the keys from STEP_USER_DATA_SCHEMA with values provided by the user.
+    """
+    return {"device_id": data["device_id"]}
```

Comment:
How does this validate uniqueness and the ability to connect?

Reply:
no, because i don't want to make people go turn on their cooker. All they need here is to copy-paste from the android app.